src/Features/Core/Portable/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.cs

```diff
@@ -23,9 +23,10 @@ internal abstract class AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer
         // The NotConfigurable custom tag ensures that user can't turn this diagnostic into a warning / error via
         // ruleset editor or solution explorer. Setting messageFormat to empty string ensures that we won't display
         // this diagnostic in the preview pane header.
+        // Setting category to "Compiler" ensures we show the unnecessary usings even in power save mode.
```

Comment:
yeah, this is such a hack.  why is remove-imports in this category, but other fixes would not be?  somthing is fishy here.

How about: in power saving, we only run for the visible file.  Other files only get fixed up when opened or get focus?

Reply:
> How about: in power saving, we only run for the visible file. Other files only get fixed up when opened or get focus?

Only run what? We want to explicit turn off any analyzers that are not required for minimal acceptable IDE experience. We can possible add a new custom tag for such analyzers.

Reply:
> Only run what? 

Only run analyzers.  However, we don't run them for any other files.  That will still help out with power savings, while not diminishing the experience for hte actively edited file.  I think users would understand and be ok with that.

Reply:
That could also be potentially expensive, if you have an expensive analyzer running. I think our preference was to only run extremely important analyzers just for active file. It is fine to create a new mechanism to identify those, but we don't want to run other analyzers.

Reply:
Gotcha.  I think this is a reasonable place to start.